repository_service_tuf/cli/admin/ceremony.py

```diff
@@ -695,7 +695,7 @@ def ceremony(context, bootstrap, file, upload, save) -> None:
                 )
 
                 if rolename == Roles.TARGETS.value:
-                    assert role.paths is not None
```

Comment:
If we don't check somehow whether `role.paths` is not `None`, `mypy` returns the following error: `repository_service_tuf/cli/admin/ceremony.py:701: error: List or tuple expected as variadic arguments  [misc]`

From `_configure_role(rolename, role)` we can see that `role.paths` will not be `None` (`role.paths = delegate_paths`), but `mypy` does not pick that up, thus with this assert we let the static typechecker know that it's not `None`. Therefore, we do not mind whether the `assert` is removed when the code is compiled (it's only used for `mypy`).

This could be checked with an `if` but you would need to handle the case and as I mentioned above I don't see this actually being a possible scenario.

Reply:
Okay, here it seems that this is a valid case for an `assert`.
We know that `role.paths` will not be `None` from `_configure_role(rolename,role)`, but we want to add this assert because of `mypy`.

That's my point in raising the question: do we need an assert here or we want to use an actual `if` statement? Asserts are great if you know that certain cases cannot happen, but you want `mypy` to be happy, otherwise, it's better to use actual `if` statements.